app/src/main/java/com/plenart/organizeme/utils/BoardDiffCallback.kt

```diff
@@ -0,0 +1,12 @@
+package com.plenart.organizeme.utils
+
+import androidx.recyclerview.widget.DiffUtil
+import com.plenart.organizeme.models.Board
+
+class BoardDiffCallback : DiffUtil.ItemCallback<Board>() {
```

Comment:
DiffCallback classes can be inside of adapter, because they will not be used outside of it.